typings/index.d.ts

```diff
@@ -1666,12 +1667,13 @@ declare module 'discord.js' {
 		private connection: WebSocket | null;
 		private helloTimeout: NodeJS.Timeout | null;
 		private eventsAttached: boolean;
+		private expectedGuilds: Set<Snowflake>;
+		private readyTimeout: NodeJS.Timeout | null;
```

Comment:
This property is either a NodeJS.Timeout or undefined: https://github.com/discordjs/discord.js/blob/e47fc1e0f85ee63409979084bd45e0b748899fad/src/client/websocket/WebSocketShard.js#L149

```suggestion
		private readyTimeout: NodeJS.Timeout | undefined;
```